src/main/java/com/google/maps/model/SpeedLimit.java

```diff
@@ -30,7 +30,7 @@
    */
   public double speedLimit;
 
-  /** Returns the speed limit in miles per hour (MPH). */
+  /** @return Returns the speed limit in miles per hour (MPH). */
```

Comment:
I've always written JavaDoc with Returns as the first word of the @return line, because the generated JavaDoc reads a tad weird without it. Or at least it did back in the 1.4 days...